android/java/org/chromium/chrome/browser/onboarding/OnboardingPrefManager.java

```diff
@@ -71,7 +71,7 @@
     private static final String BING = "Bing";
     private static final String STARTPAGE = "Startpage";
     private static final String YAHOO = "Yahoo";
-    private static final String YANDEX = "Yandex";
+    public static final String YANDEX = "Yandex";
```

Comment:
not sure why it has been changed from `private` to `public`

Reply:
we had to explicitly make yandex as default browser so making the short name as public static.